sections/model_overview/main.tex

```diff
@@ -3,43 +3,48 @@ \section{Overview of game theoretic model} \label{sec:model_overview}
+while the EMS aims to minimise the time that its ambulances 
 are blocked.
-Although we generalise for any such system, relating this back to the
-motivating case study, here the decision makers of the queueing systems
-represent the ED managers and the distributor is the EMS who allocate (send)
-ambulances to one of the EDs (see Section \ref{sec:ems_ed_application}). 
-% TODO: Depending on how this is described earlier, we might need a little more 
-% here (as to what we mean by "blocked")
+This can be generalised for any such system where instead of EDs there are some
+queueing systems and instead of the EMS there is some distributor that allocates 
+individuals to the queueing systems. 
 
-The queueing systems are designed in such a way where they can accept two types
+The queueing systems of the hospitals are designed in such a way where they can
+accept two types
 of individuals. 
-These are the individuals that are allocated by the distributor and 
-other individuals from other sources. 
-Each queueing system may then choose to block the individuals that arrive from 
-the distributor when the system reaches a certain capacity. 
-The strategy sets for each queueing system is the set 
+% TODO: Decide whether to define type 1, 2 only in game section, only in queueing section, or both
+Type 2 individuals are defined as non-urgent patients arriving in ambulances 
+that are allocated by the ambulance service and 
+type 1 individuals are individuals arriving from other sources 
+(e.g walk-in patients, urgent patients from either walk-ins or ambulances).
+Each hospital may then choose to block type 2 individuals 
+when the hospital reaches a certain capacity. 
+The strategy sets for each hospital is the set 
 \( \{T \in \mathbb{N} \;|\; 1 \leq T \leq N\} \) where \(N \in\{N_A, N_B\}\) are 
-the total capacities of the two queueing systems. We denote the chosen actions 
+the total capacities of hospitals \(A\) and \(B\). We denote the chosen actions 
 from the strategy set as \(T_A, T_B\) and call these \textit{threshold}s.
 
-Both queueing systems follow a queueing model that has two waiting zones for 
+Both hospitals follow a queueing model with two waiting zones for 
 individuals. 
-The first waiting zone is where the individuals queue right before receiving 
-their service and has a capacity of \( N - C \), where \(N\) is the total 
-capacity of the waiting space and \(C\) is the number of servers. 
-The second waiting zone is where the individuals, that are sent from the 
-distributor, stay until they are allowed to enter the first waiting zone.
-The second waiting zone has a capacity of \(M\) and no servers.
+The first waiting zone (i.e. the waiting space of the hospital) is where the 
+patients queue right before receiving 
+their service and has a queue capacity of \( N - C \), where \(N\) is the total 
+capacity of the hospital and \(C\) is the number of servers. 
+% TODO: Mention what we mean by servers here?
```

Comment:
```suggestion
capacity of the hospital and \(C\) is the number of healthcare 
professionals able to see them.
```

I'm not convinced by my suggestion.

Reply:
Will include this and add a `TODO` to see how well it works after restructuring